### PR TITLE
fix typo in iperf_util.c

### PR DESCRIPTION
**Pull request by iZarrios**
fix typo in `make_cookie` documentation.

### CODE REVIEW

File: src/iperf_util.c [hunk 1]
```diff
@@ -105,7 +105,7 @@ void fill_with_repeating_pattern(void *out, size_t outsize)
  * Generate and return a cookie string
  *
  * Iperf uses this function to create test "cookies" which
- * server as unique test identifiers. These cookies are also
+ * serves as unique test identifiers. These cookies are also
```
Comment (bmah888) [inline src/iperf_util.c:108]: If we're going to fix that word, it should be just "serve" because the word "cookies" is plural.

**Review by bmah888 — CHANGES_REQUESTED** [review]
Thanks for the pull request! Definitely we want good documentation that isn't confusing. As long as we're fixing this word it'd be good to get the subject/verb agreement correct.

**Comment by iZarrios** [comment]
Fixed it.

**Review by bmah888 — APPROVED** [review]

**Comment by bmah888** [comment]
Thanks for the revision, merged!

**Pull request merged**